pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation.adoc

```diff
@@ -0,0 +1,41 @@
+[[apim-kubernetes-operator-installation]]
+= GKO installation and deployment
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_installation.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+You can deploy the Gravitee Kubernetes Operator [GKO] in two ways:
+
+1. On a production Kubernetes cluster.
+2. On a local Kubernetes cluster (for testing purposes only).
+
+
+== Deploying GKO on a production cluster
+
+Deploying the Gravitee Kubernetes Operator (GKO) on an existing remote / cloud-based APIM-ready link:https://k3d.io/[k3d^] Kubernetes cluster involves the following stages:
+
+1. Deploy the GKO on your cluster.
+2. Create a Management Context custom resource.
+3. Create an API Definition custom resource - this creates a new API on the cluster.
+4. Test the new API by calling it through the APIM Gateway.
+
+Read all about it link:{{ '/apim/3.x/apim_kubernetes_operator_installation_cluster.html' | relative_url }}[here].
+
+== Local setup and deployment for testing purposes
+
+You can try out the Gravitee Kubernetes Operator on your local machine by running the operator locally against an APIM-ready link:https://k3d.io/[k3d^] cluster. This setup involves the following stages:
+
+1. Set up your environment (prerequisite step).
+2. Create a local APIM-ready k3d Kubernetes cluster.
+3. Check if the Gravitee CRDs are available on your cluster.
```

Comment:
🤔 I don't think this is the right order. 

Gravitee CRDs was installed by this step `6. Deploy the GKO on the new cluster.` so you can only find CRDs after

Reply:
Updated!